modules/logging-loki-storage-aws.adoc

```diff
@@ -25,3 +25,31 @@ $ oc create secret generic logging-loki-aws \
   --from-literal=access_key_secret="<aws_access_key_secret>" \
   --from-literal=region="<aws_region_of_your_bucket>"
 ----
+
+== AWS storage for STS enabled clusters
+
+You can create a secret manually by running the following command:
+[source,terminal,subs="+quotes"]
+----
+oc -n openshift-logging create secret generic "logging-loki-s3"
+--from-literal=bucketnames="<s3_bucket_name>"
+--from-literal=region="<bucket_region>"
+----
+
+If your cluster has STS enabled, the Cloud Credential Operator (CCO) supports short-term authentication using AWS tokens. Secrets created by using the CCO utility (`ccoctl`) use a different format to authenticate.
+
+.Example AWS STS secret
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: <target-namespace>
+  name: <target-secret-name>
+stringData:
+  credentials: |-
+    [default]
+    sts_regional_endpoints = regional
+    role_arn: <operator-role-name>
+    web_identity_token_file: <path-to-token>
+----
```

Comment:
The secret generated by CCO on the cluster (from the `CredentialsRequest` that the operator creates) is just used internally. The "LokiStack secret" mentioned above (with `bucketnames` and `region`) is the only configuration relevant to the user.